static/css/stylesheet.css

```diff
@@ -326,6 +326,11 @@ Full-Width Styles
   zoom: 1;
 }
 
+.oppia-foundation-no-horizontal-padding {
+  padding-left: 0px;
```

Comment:
Prefer 0 to 0px (remove units when it's 0).

Reply:
Thanks for spotting this :)